pkg/testhelper/testhelper.go

```diff
@@ -27,6 +31,90 @@ const (
 	ERROR
 )
 
+type ObjectOut struct {
+	ObjectType   string
+	ObjectFields map[string]string
+}
+
+type FailureReasonOut struct {
+	CompliantObjectsOut    []*ObjectOut
+	NonCompliantObjectsOut []*ObjectOut
+}
+
+const (
+	Namespace              = "Namespace"
+	PodName                = "PodName"
+	ContainerName          = "ContainerName"
+	ProcessID              = "ProcessID"
+	ProcessCommandLine     = "ProcessCommandLine"
+	SchedulingPolicy       = "SchedulingPolicy"
+	SchedulingPriority     = "SchedulingPriority"
+	ReasonForNonCompliance = "ReasonForNonCompliance"
+	ReasonForCompliance    = "ReasonForCompliance"
+	Category               = "Category"
+)
+
+const (
+	UndefinedType        = "Undefined Type"
+	CnfType              = "Cnf"
+	PodType              = "Pod"
+	ContainerType        = "Container"
+	ContainerProcessType = "ContainerProcess"
+)
+
+func NewProcessObjectOut(aContainer *ObjectOut, aPolicy, aPriority, aCommandLine string) (out *ObjectOut) {
```

Comment:
Although I don't like the fact that this function can only be used in test cases that deal with process scheduling policies, it looks like a good candidate to make a it a struct's method:
```
func (object *ObjectOut) SetContainerProcessValues(policy, priority, commandLine string) (*ObjectOut) {
   ...
```
Thus, using my name suggestion on line 96 for the base function, you could do:
```
containerProcessReport := NewTestTargetReport(ContainerProcessType, false).SetContainerProcessValues("SCHED_OTHER", 0, "...")
```
or
```
containerProcessReport := NewTestTargetReport(ContainerProcessType, false)
containerProcessReport.SetContainerProcessValues("SCHED_OTHER", 0, "...")
```

Reply:
Fixed.